monai/metrics/utils.py

```
@@ -244,6 +267,7 @@ def get_surface_distance(
     seg_gt: NdarrayOrTensor,
     distance_metric: str = "euclidean",
     spacing: int | float | np.ndarray | Sequence[int | float] | None = None,
+    mask: NdarrayOrTensor | None = None,
```

Ruff (0.15.5)

[warning] Unused function argument: `mask`

(ARG001)